packages-exp/auth-exp/src/core/auth/auth_impl.ts

```diff
@@ -203,6 +203,10 @@ export class AuthImpl implements Auth, _FirebaseService {
       appName: this.name
     });
 
+    assert(/^https?:\/\//.test(url), AuthErrorCode.INVALID_EMULATOR_SCHEME, {
```

Comment:
FWIW This just throws `ARGUMENT_ERROR` in the JS SDK. I don't think we need a dedicated error code for it.

Reply:
We don't do message overrides to errors in the new SDK, so in order to have that text it needs to be its own error code (otherwise it's just going to be a plain argument error with no explanation)

Reply:
Resolving this